samples/inputs/button-group/overview/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "react-button-group-overview",
+  "description": "This project provides example of Button Group Overview using Infragistics React components",
+  "author": "Infragistics",
+  "homepage": ".",
+  "version": "1.4.0",
+  "private": true,
+  "scripts": {
+    "start": "set PORT=4200 && react-scripts --max_old_space_size=10240 start",
+    "build": "react-scripts --max_old_space_size=10240 build ",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "dependencies": {
+    "igniteui-dockmanager": "1.14.2",
+    "igniteui-react": "18.6.0",
+    "igniteui-react-core": "18.6.0",
```

Comment:
Update igniteui-react and igniteui-react-core package version to 18.6.1-alpha.0 and igniteui-dockmanager package version to 1.14.3 for every sample to be up to date.